api/stream.go

```diff
@@ -50,6 +50,8 @@ func configGinStreamRestRouter(router *gin.Engine, daoWrapper dao.DaoWrapper) {
 				thumbs.GET(":fid", routes.getThumbs)
 				thumbs.GET("/live", routes.getLiveThumbs)
 				thumbs.GET("/vod", routes.getVODThumbs)
+				thumbs.POST("/customlive", routes.putCustomLiveThumbnail)
```

Comment:
Imo the logical method and path would be something like `POST /api/stream/:streamID/thumbnails`
```suggestion
				thumbs.POST("/", routes.putCustomLiveThumbnail)
```